doc/realtime_servo/src/servo_cpp_interface_demo.cpp

```diff
@@ -97,7 +97,7 @@ int main(int argc, char** argv)
 {
   rclcpp::init(argc, argv);
   rclcpp::NodeOptions node_options;
-  node_options.use_intra_process_comms(true);
+  node_options.use_intra_process_comms(false);
```

Comment:
Could you add a short note why this is necessary?

Reply:
I updated this, lmk what you think of my note.